Gruntfile.js

```diff
@@ -53,34 +54,26 @@ const bedrockHeadless = (tests, browser, auto, opts) => {
         name: 'headless-tests',
         browser,
         testfiles: testFolders(tests, auto),
-
-        // we have a few tests that don't play nicely when combined together in the monorepo
-        retries: 3,
         ...opts
       }
     }
   }
 };
 
-const bedrockBrowser = (tests, browserName, osName, bucket, buckets, chunk, remote, auto, opts) => {
+const bedrockBrowser = (tests, browserName, osName, bucket, buckets, chunk, auto, opts) => {
   const name = opts.name ? opts.name : `${browserName}-${osName}`;
   if (tests.length === 0) {
     return {};
   } else {
     return {
       browser: {
         ...bedrockDefaults,
-        overallTimeout: 3600000,
         name: name,
         browser: browserName,
         testfiles: testFolders(tests, auto),
         bucket: bucket,
         buckets: buckets,
         chunk: chunk,
-        remote: remote,
-
-        // we have a few tests that don't play nicely when combined together in the monorepo
-        retries: 3,
```

Comment:
So we getting rid of this one, I guess it makes sense since it's been an escape hatch also it has caused confusion when the test say passes on remote but not locally since locally it doesn't retry.